src/exec.js

```diff
@@ -109,6 +110,7 @@ function execSync(cmd, opts, pipe) {
 function execAsync(cmd, opts, pipe, callback) {
   opts = common.extend({
     silent: common.config.silent,
+    fatal: common.config.fatal,
```

Comment:
This (and the line above it) seem redundant because we assign to this variable inside `_exec()`. I guess it's fine since `silent` left this, but could you leave a `TODO(nfischer)` to clean this up?

Reply:
Left a TODO comment for you on each of these lines. Let me know if you want it done another way.

Reply:
That TODO looks good, thanks